server/index.js

```diff
@@ -19,25 +25,30 @@ const authRouter = require("./src/routes/auth");
 const userRouter = require("./src/routes/user");
 const clientRouter = require("./src/routes/client");
 
+// Set up cors for local dev connection with frontend
+app.use(cors(corsOption));
+
 // Set up session for authorization
 app.use(
   session({
     secret: process.env.SESSION_SECRET,
     resave: false,
     saveUninitialized: false,
     store: new SQLiteStore({ db: "sessions.db", dir: "./var/db" }),
-  }),
+  })
 );
 app.use(passport.authenticate("session"));
 
 app.get("/", (req, res) => {
-  res.send("Helloooo World!");
-});
-
-app.listen(port, () => {
-  console.log(`Example app listening on port ${port}`);
+  res.send("Helloooo World! Hi :)");
 });
```

Comment:
This was sort of a dummy request I set up when creating the repo, it can be deleted 😄